linode/manager
# M3-1722 Boot into different config profiles

This PR fixes a bug that prevented users from booting a linode into a specific configuration profile.

The solution was to pass in a `config_id` argument to the existing `linodeBoot` API method. 

### To test:

**Steps to reproduce bug:** 

_Prerequisites:_

- Have a Linode with at least two configuration profiles with different distros. In this example we’ll use CentOS and Ubuntu. 
- Have `develop` branch checked out.

_Steps:_

1) Power down linode
2) Power on linode and use CentOS config profile
3) SSH into the linode
4) Enter `cat /etc/os-release | grep ^NAME=`
5) Observe: the terminal will print `NAME="CentOS Linux”`
6) Power down linode
7) Power on linode and use Ubuntu config profile
8) SSH into the linode
9) Enter `cat /etc/os-release | grep ^NAME=`
10) Observe: the terminal will AGAIN print `NAME=“CentOS Linux”`

**To verify bug fix:** 

1) Checkout this branch.
2) Repeat steps 1-9 of the previous section.
3) Observe: the terminal will print `NAME="Ubuntu”`
